tests/tools/oecert/host/host.cpp

```diff
@@ -196,9 +201,59 @@ static void _display_help(const char* cmd)
     printf("\t%s\n", INPUT_PARAM_OPTION_VERBOSE);
 }
 
+// Get full path of oecert running executable, then get enclave filename by:
+// In linux, replace "<path>/oecert" with "<path>/oecert_enc.signed"
+// In windows, replace "<path>/oecert.exe" with "<path>/oecert_enc.signed"
```

Comment:
Does Windows enclave file not have .DLL extension?

Reply:
Just checked windows build and it didn't have DLL extension. So do other testing.